pkg/textutil/textutil.go

```diff
@@ -17,3 +22,24 @@ func ExecuteTemplate(tmpl string, args interface{}) ([]byte, error) {
 	}
 	return b.Bytes(), nil
 }
+
+// TemplateFuncMap is a text/template FuncMap.
+var TemplateFuncMap = template.FuncMap{
+	"json": func(v interface{}) string {
+		var b bytes.Buffer
+		enc := json.NewEncoder(&b)
+		enc.SetEscapeHTML(false)
+		if err := enc.Encode(v); err != nil {
+			panic(fmt.Errorf("failed to marshal as JSON: %+v: %w", v, err))
+		}
+		return strings.TrimSpace(b.String())
```

Comment:
Why is the `TrimSpace` needed?

Reply:
Because otherwise it produces "\n\n"-delimited JSON lines

Reply:
Wouldn't it be clearer to show the intent with this:

```suggestion
		return strings.TrimSuffix(b.String(), "\n")
```

Reply:
done